packages/pass-style/src/passStyleOf.js

```diff
@@ -65,6 +67,11 @@ const makeHelperTable = passStyleHelpers => {
  * @returns {PassStyleOf}
  */
 const makePassStyleOf = passStyleHelpers => {
+
+  if (typeof globalThis?.VatData?.passStyleOf === 'function') {
+    return globalThis.VatData.passStyleOf;
+  }
+
```

Comment:
Why put this check inside the maker instead of the actual `passStyleOf` exported below? I know it'd become a less readable ternary, but it just feels strange to have a maker return a preexisting value, ignoring it's parameter.

Reply:
I will not be bothered if this becomes a cascade of the two namespaces whatever they will be in time. It would be a very small scar.

Reply:
>  I know it'd become a less readable ternary,

I wasn't bad. Done.